egs/csj/s5/local/nnet/run_dnn_tandem_uc.sh

```diff
@@ -0,0 +1,283 @@
+#!/bin/bash
```

Comment:
This type of script is very old, I'd prefer not to support it. A newer and simpler version is here (it has less stages and drops the second stream of PLPs): https://github.com/vesis84/kaldi/blob/master/egs/swbd/s5c/local/nnet/run_dnn_tandem_uc.sh